doc/reference/ubuntu-image.rst

```diff
@@ -0,0 +1,246 @@
+ubuntu-image syntax and options
+===============================
+
+``ubuntu-image`` is a program for generating bootable disk images. It supports building snap_-based and classical preinstalled Ubuntu images.
```

Comment:
Should we add a link somewhere on this page to the snap store page of the tool? Or to instructions on how to install it?